src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -867,7 +867,7 @@ void Transformations::MainSnippets(void) {
 #if defined(OPENVINO_ARCH_X86_64)
     auto is_supported_matmul = [this](const std::shared_ptr<const ov::Node>& n) {
         const auto matmul = ov::as_type_ptr<const ov::op::v0::MatMul>(n);
-        if (!matmul)
+        if (!matmul || matmul->is_dynamic())
```

Comment:
Do I understand correctly that the tokenization tets we discussed offline can now be reproduced on this PR if we revert this change?

Reply:
You're right!
But if dynamic Brgemm supports only fp32 in CPU, please move this condition on L878